lib/file_transfer_agent/encryption_types.js

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
do we need the configuration to be in a separate file? someone may use exports of file in applications?

Reply:
I initially thought it would be good to extract this since this file was already getting big, but it doesn't really make any difference.I moved it back to encryption_util.js